From 39f0266f3326a445b7700310234841e803e78e76 Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Sun, 14 Aug 2022 00:10:32 -0400
Subject: [PATCH 1/1] enable-testing-on-arm

For some reason, testing suite for arm is disabled at build.
Patch reenables build of testing suite

---
 src/libpsl-native/CMakeLists.txt | 15 +++++----------
 1 file changed, 5 insertions(+), 10 deletions(-)

diff --git a/src/libpsl-native/CMakeLists.txt b/src/libpsl-native/CMakeLists.txt
index e4f33cc..61abd96 100644
--- a/src/libpsl-native/CMakeLists.txt
+++ b/src/libpsl-native/CMakeLists.txt
@@ -13,13 +13,8 @@ endif()
 
 set(LIBRARY_OUTPUT_PATH "${PROJECT_SOURCE_DIR}/../powershell-unix")
 
-if (CMAKE_SYSTEM_PROCESSOR MATCHES "arm*")
-    message(STATUS "Building for ARM, no tests")
-    add_subdirectory(src)
-else ()
-    # test in BUILD_DIR
-    message(STATUS "Tests enabled")
-    enable_testing()
-    add_subdirectory(src)
-    add_subdirectory(test)
-endif ()
+# test in BUILD_DIR
+message(STATUS "Tests enabled")
+enable_testing()
+add_subdirectory(src)
+add_subdirectory(test)
-- 
2.36.2

